npm/private/npm_import.bzl

```diff
@@ -186,25 +182,7 @@ _LINK_JS_PACKAGE_LIFECYCLE_TMPL = """\
 _LINK_JS_PACKAGE_LINK_IMPORTED_STORE_TMPL = """\
 # Generated npm_package_store and npm_link_package_store targets for npm package {package}@{version}
 # buildifier: disable=function-docstring
-def npm_link_imported_package_store(name):
```

Comment:
This macro is generated for each package, then invoked for each instance of that package. Then each invocation inspected the shared data to find which invocation it was.

Instead each invocation can just pass the data required.